hub/api/tests/test_api.py

```diff
@@ -473,6 +474,7 @@ def test_hub_dataset_suffix_bug(hub_cloud_ds, hub_cloud_dev_token):
 
     # need to delete because it's a different path (won't be auto cleaned up)
     ds.delete()
+    hub_cloud_ds.delete()
```

Comment:
does the [hub cloud path fixture](https://github.com/activeloopai/Hub/blob/9ec4a0021950a1973dd4374569ebd308d0235fdd/hub/tests/path_fixtures.py#L139) not cover this?

Reply:
It should but in this case dataset remains in hub. This is a temporary solution to not fill the platform with unused test datasets.

Reply:
it should only remain if you ctrl+c before the test finishes. if i'm wrong, can we find a way to fix this with the fixtures? it will help us in the long run